src/main/java/Duke.java

```diff
@@ -1,10 +1,151 @@
+    private static final String COMMAND_ADD_DEADLINE = "deadline";
+    private static final String COMMAND_ADD_EVENT = "event";
+
+    private static Task[] taskItems = new Task[MAX_ITEMS];
+    private static int taskItemCount = 0;
+
+    public static void printAddTaskItems() {
+        String printAddItem = TOP_DIVIDER + ADD_TASK_ITEM_DESC + taskItems[taskItemCount]
+                + CUR_NO_OF_TASK_START_DESC + (taskItemCount + 1) + CUR_NO_OF_TASK_END_DESC + BOTTOM_DIVIDER;
+        System.out.print(printAddItem);
+    }
+
+    public static void addDefaultTaskItems(String taskName) {
+        taskItems[taskItemCount] = new Task(taskName);
+        printAddTaskItems();
+        taskItemCount++;
+    }
+
+    public static void addDeadlineTaskItems(String taskName, String by) {
+        taskItems[taskItemCount] = new Deadline(taskName, by);
+        printAddTaskItems();
+        taskItemCount++;
+    }
+
+    public static void addEventTaskItems(String taskName, String from, String to) {
+        taskItems[taskItemCount] = new Event(taskName, from, to);
+        printAddTaskItems();
+        taskItemCount++;
+    }
+
+    public static void addTodoTaskItems(String taskName) {
+        taskItems[taskItemCount] = new Todo(taskName);
+        printAddTaskItems();
+        taskItemCount++;
+    }
+
+    public static void markTaskItems(int taskItemNo, String command) {
+        if (command.equals(COMMAND_MARK_TASK)) {
+            taskItems[taskItemNo].setCompleted();
+            String printTaskCompleted = TOP_DIVIDER + MARK_TASK_DESC + taskItems[taskItemNo] + BOTTOM_DIVIDER;
+            System.out.print(printTaskCompleted);
+        } else {
+            taskItems[taskItemNo].setIncomplete();
+            String printTaskIncomplete = TOP_DIVIDER + UNMARK_TASK_DESC + taskItems[taskItemNo] + BOTTOM_DIVIDER;
+            System.out.print(printTaskIncomplete);
+        }
+    }
+
+    public static void listTaskItems() {
+        System.out.print(TOP_DIVIDER + LIST_TASK_ITEM_DESC);
+
+        for (int itemNo = 0; itemNo < taskItemCount; itemNo++) {
+            String printItem = WHITESPACE + (itemNo + 1) + DOT + taskItems[itemNo] + System.lineSeparator();
+            System.out.print(printItem);
+        }
+
+        System.out.print(LIST_TASK_ITEM_BOTTOM_DIVIDER);
+    }
+
+    public static void printWelcomeMessage() {
+        String printGreet = TOP_DIVIDER + WELCOME_MESSAGE + BOTTOM_DIVIDER;
+        System.out.print(printGreet);
+    }
+
+    public static void printExitMessage() {
+        String printExit = TOP_DIVIDER + EXIT_DESC + BOTTOM_DIVIDER;
+        System.out.print(printExit);
+    }
+
+    public static String getInput() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        return line;
+    }
+
+    public static void processInput(String input) {
+        String[] words = input.split(" ", 2);
+        String command = words[0];
+
```

Comment:
I like how you followed the coding standards to name your methods and variables in camelCase!